client/src/components/restaurantMaps/BlissBeeMap.jsx

```diff
@@ -0,0 +1,80 @@
+// imports for map and styling
```

Comment:
You shouldn't need these files in addition to your normal Map component. However these work great as is 👍